class.jetpack-debugger.php

```diff
@@ -60,6 +81,26 @@ public static function jetpack_debug_display_handler() {
 		$debug_info .= "\r\n" . esc_html( "SITE_URL: " . site_url() );
 		$debug_info .= "\r\n" . esc_html( "HOME_URL: " . home_url() );
 
+		$debug_info .= "\r\n";
+		require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-modules.php';
+		$sync_module = Jetpack_Sync_Modules::get_module( 'full-sync' );
+
+		$debug_info .= "\r\n". esc_html( "Jetpack Sync Full Status: " . print_r( $sync_module->get_status(), 1 ) );
+
+		$next_schedules = wp_next_scheduled( 'jetpack_sync_full' );
+		if( $next_schedules ) {
+			$debug_info .= "\r\n". esc_html( "Next Jetpack Full Sync Schedule: " . date( 'r', $next_schedules ) );
```

Comment:
Could you make all the strings translatable, so users can understand the debug results as well?

Thanks!